sense/loading.py

```diff
@@ -55,6 +55,16 @@ def __init__(self, model_name: str, version: str, feature_converters: List[str])
 
         self.feature_converters = feature_converters
 
+    def model_exist(self):
+        model_weights = MODELS[self.model_name][self.version]
+        path_weights = {name: model_weights[name] for name in ['backbone'] + self.feature_converters}
+        files_exist = all(os.path.exists(prepend_resources_path(path)) for path in path_weights.values())
```

Comment:
This code looks almost identical to the checks below in `get_weights`. Can you manage to reuse it there?